src/parquet/file/file-serialize-test.cc

```diff
@@ -96,6 +96,11 @@ class TestSerialize : public ::testing::Test {
         values_out.data(), &values_read);
     ASSERT_EQ(100, values_read);
     ASSERT_EQ(values, values_out);
+
+    auto stats = std::static_pointer_cast<Int64Statistics>(rg_reader->GetColumnStats(0));
+    ASSERT_EQ(0, stats->NullCount());
+    ASSERT_EQ(128, stats->Min());
+    ASSERT_EQ(128, stats->Max());
```

Comment:
This is a fairly weak test case because there's no variety to the input data. Relates to the question around verifying the correctness of the comparators